app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/launcher/LauncherContract.kt

```diff
@@ -0,0 +1,16 @@
+package org.mifos.mobile.cn.ui.mifos.launcher
+
+import org.mifos.mobile.cn.ui.base.MvpView
+
+/**
+ * @author Manish Kumar
+ * On 31/05/18.
+ */
+
+interface LauncherContract {
```

Comment:
remove this file we don't have any logic to write for now if you would have in future we will add it.